modules/specifying-host-groups-vsphere.adoc

```diff
@@ -0,0 +1,230 @@
+  template:
+    machineType: machines_v1beta1_machine_openshift_io
+    machines_v1beta1_machine_openshift_io:
+      failureDomains:
+        platform: VSphere
+        vsphere:
+        - name: <failure_domain_1_name>
+        - name: <failure_domain_2_name> 
+----
+
+.. Verify that your control plane nodes have finished updating before proceeding further. To do this, run the following command:
++
+[source,terminal]
+----
+$ oc get controlplanemachinesets.machine.openshift.io -n openshift-machine-api
+----
+
+. Create new `MachineSet` custom resources for your failure domains.
+
+.. To retrieve the configuration of an existing `MachineSet` custom resource for use as a template, run the following command:
++
+[source,terminal]
+----
+$ oc get machinesets.machine.openshift.io -n openshift-machine-api <existing_machine_set> -o yaml > machineset-<failure_domain_name>.yaml
+----
+
+.. Copy the template as needed to create .yaml `MachineSet` custom resource files for each failure domain you defined in your `infra.yaml` file. Refer to the following example:
++
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+  name: <machineset_name>
+  namespace: openshift-machine-api
+spec:
+  replicas: 0
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+      machine.openshift.io/cluster-api-machineset: <machineset_name>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <machineset_name>
+    spec:
+      lifecycleHooks: {}
+      metadata: {}
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1beta1
+          credentialsSecret:
+            name: vsphere-cloud-credentials
+          diskGiB: <disk_GiB>
+          kind: VSphereMachineProviderSpec
+          memoryMiB: <memory_in_MiB>
+          metadata:
+            creationTimestamp: null
+          network:
+            devices:
+            - networkName: VM Network
+          numCPUs: <number_of_cpus>
+          numCoresPerSocket: <number_of_cores_per_socket>
+          snapshot: ""
+          template: <template_name>
+          userDataSecret:
+            name: worker-user-data
+          workspace:
+            datacenter: <data_center_1>
+            datastore: /<data_center_1>/datastore/<datastore_1>
+            folder: /<data_center_1>/vm/<folder>
+            resourcePool: /<data_center_1>/host/<cluster_1>/Resources
+            server: <server_ip_address>
+            vmGroup: <name_of_vm_group_1>
+----
+
+.. For each `MachineSet` custom resource file, run the following command:
++
+[source,terminal]
+----
+$ oc create -f <name_of_machine_set_file>.yaml
```

Comment:
Might be obvious but providing information to replace `<name_of_machine_set_file>.yaml` with the name of your file can be useful.